packages/react/src/components/FocusTrapZone/FocusTrapZone.test.tsx

```diff
@@ -193,32 +194,37 @@ describe('FocusTrapZone', () => {
     it('can tab across FocusZones with different button structures', async () => {
       expect.assertions(3);
 
-      const topLevelDiv = (ReactTestUtils.renderIntoDocument(
-        <div onFocusCapture={_onFocus}>
-          <FocusTrapZone forceFocusInsideTrap={false} className={ftzClassname}>
-            <FocusZone direction={FocusZoneDirection.horizontal} data-is-visible={true}>
-              <div data-is-visible={true}>
-                <button className="a">a</button>
-              </div>
-              <div data-is-visible={true}>
-                <button className="b">b</button>
-              </div>
-              <div data-is-visible={true}>
-                <button className="c">c</button>
-              </div>
-            </FocusZone>
-            <FocusZone direction={FocusZoneDirection.horizontal} data-is-visible={true}>
-              <div data-is-visible={true}>
+      const { removeTestContainer, testContainer: topLevelDiv } = createTestContainer();
```

Comment:
**suggestion:** can we use `testContainer` directly? I know previously it was named `topLevelDiv`, but IMO it doesn;t make the code more self-explanatory.

with that we can get rid of using aliases within destructuring. which should be avoided for sake of readability from my experience.

WDYT ?